exdb/tests/integration_tests.py

```diff
@@ -299,12 +316,12 @@ class ExperienceApprovalViewTest(StandardTestCase):
     def test_gets_correct_experience(self):
         e = self.create_experience('pe')
         self.create_experience('pe')
-        response = self.anon_client.get(reverse('approval', args=str(e.pk)))
+        response = self.clients['ra'].get(reverse('approval', args=str(e.pk)))
```

Comment:
Same here for the Approval view tests, we should probably use the Hall-Staff client to do the approvals.